keep-ui/app/alerts/alert-table-tab-panel.tsx

```diff
@@ -23,26 +23,32 @@ const getPresetAlerts = (alert: AlertDto, presetName: string): boolean => {
 };
 
 const getOptionAlerts = (alert: AlertDto, options: Option[]): boolean =>
-  options.every((option) => {
-    const [key, value] = option.value.split("=");
-
-    if (key && value) {
-      const lowercaseKey = key.toLowerCase() as keyof AlertDto;
-      const lowercaseValue = value.toLowerCase();
-
-      const alertValue = alert[lowercaseKey];
-
-      if (Array.isArray(alertValue)) {
-        return alertValue.every((v) => lowercaseValue.split(",").includes(v));
-      }
-
-      if (typeof alertValue === "string") {
-        return alertValue.toLowerCase().includes(lowercaseValue);
-      }
-    }
-
-    return true;
-  });
+  options.length > 0
+    ? options.some((option) => {
+        const [key, value] = option.value.split("=");
+
+        if (key && value) {
+          const attribute = key.toLowerCase() as keyof AlertDto;
+          const lowercaseAttributeValue = value.toLowerCase();
+
+          const alertAttributeValue = alert[attribute];
+
+          if (Array.isArray(alertAttributeValue)) {
+            return alertAttributeValue.every((v) =>
+              lowercaseAttributeValue.split(",").includes(v)
+            );
+          }
+
+          if (typeof alertAttributeValue === "string") {
+            return alertAttributeValue
+              .toLowerCase()
+              .includes(lowercaseAttributeValue);
+          }
+        }
+
+        return true;
+      })
+    : true;
```

Comment:
Does this change do anything different from the original?

Reply:
@talboren

Reply:
Yep! `every` to `some` (meaning that the filters are `OR` based). We will improve this over time.